util/copycmd.bat

```diff
@@ -0,0 +1,6 @@
+REM @echo off
+set "PATH=C:\windows\system32;C:\windows;C:\windows\System32\Wbem"
```

Comment:
Why do we need to set the path to explicitly include `system32` here? Maybe a comment would help understanding.

Reply:
We are resetting rather than augmenting the path, so need to set it to enough for our purposes. In this case it is enough to run robocopy. The epics path is very long so we have to do this in end_of_run.cmd too otheriwse we get a corrupt path